test cases/vala/28 generated ui file/meson.build

```diff
@@ -0,0 +1,34 @@
+project('demo', 'c', 'vala')
+
+gnome = import('gnome')
```

Comment:
```
gnome = import('gnome', required: false)
if not gnome.found()
  error('MESON_SKIP_TEST: gnome module not supported')
endif
```

This will help muon/meson++ maintainers when they import tests.

Reply:
Saves people work in the future